aip/general/0121.md

```diff
@@ -81,6 +81,27 @@ In an API with a stateless protocol, the server has the responsibility for
 persisting data, which may be shared between multiple clients, while clients
 have sole responsibility and authority for maintaining the application state.
 
+### Cyclic References
+
+The relationship between resources, such as parent-child or
+[resource references][], **must** be representable via an acylic graph.
+
+A cyclic relationship between resources increases the complexity of managing
+resources significantly more complex. Consider a resource A and B that refer to
```

Comment:
Don't need the extra "significantly more complex" at the end (likely an artifact of editing 😄 ).
```suggestion
resources. Consider a resource A and B that refer to
```